src/styles/_header.scss

```diff
@@ -37,5 +37,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 300ms;
+
+    &:hover {
+      color: $first-color;
+      cursor: pointer;
```

Comment:
`cursor: pointer;` should be in `&__link` not in the hover

Reply:
Of course! I wasn't attentive enough. Thank you